setup.cfg

```diff
@@ -40,6 +40,8 @@ dev =
     pytest-cov
     sphinx
     sphinx-rtd-theme
+z3 =
+    z3-solver
```

Comment:
See my comment on Slack, but the `test.yml` file also needs to be updated with this dependency change.